src/main/java/com/minecolonies/blockout/util/Localization.java

```diff
@@ -0,0 +1,54 @@
+package com.minecolonies.blockout.util;
+
+import net.minecraft.client.resources.I18n;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Utility class for localization support.
+ */
+public class Localization
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2974)